olittwhmcs/serializer.py

```diff
@@ -188,6 +189,20 @@ def order_domain_request_parameters(client_id, domain, payment_method,
     return parameters
 
 
+def order_bulk_products_request_parameters(parameters, **kwargs):
+    """
+    Retrieve parameters for the domain order request.
+    :param parameters: dict of the order placed.
+    :param kwargs: (Optional) Other parameters to add to the order payload.
+        Eg promo_code, affiliate_id, price (override), ...
+    :return: payload for the order product request
+    :rtype: Dictionary
+    """
+    updated_parameters = get_default_parameters()
```

Comment:
`updated_parameters` ought to be named as `default_parameters` then `parameters.update(default_parameters)`